Mobile Buy SDK/Mobile Buy SDK/Utils/BUYApplePayHelpers.m

```diff
@@ -51,11 +51,6 @@ @interface BUYApplePayHelpers ()
 
 @implementation BUYApplePayHelpers
 
-- (instancetype)initWithClient:(BUYClient *)client checkout:(BUYCheckout *)checkout
-{
-	return [self initWithClient:client checkout:checkout shop:nil];
-}
-
 - (instancetype)initWithClient:(BUYClient *)client checkout:(BUYCheckout *)checkout shop:(BUYShop *)shop
```

Comment:
Can we make these properties (client, checkout, shop) `readonly` and assign them via instance variable in the initializer? Example: `_shop = shop`.